proto/minder/v1/minder.proto

```diff
@@ -1282,17 +1282,22 @@ message RuleType {
                     string action = 2;
                     // the content of the file
                     string content = 4;
-                    // the GIT mode of the file. Not UNIX mode! String because the GH API also uses strings
-                    // the usual modes are: 100644 for regular files, 100755 for executable files and
-                    // 040000 for submodules (which we don't use but now you know the meaning of the 1 in 100644)
+                    // The UNIX file permissions to set on the file
                     optional string mode = 3;
                 }
 
                 // the title of the PR
                 string title = 1;
                 // the body of the PR
                 string body = 2;
-                repeated Content contents = 3;
+                repeated Content contents = 4;
+                // the method to use to create the PR. For now, these are supported:
+                // -- minder.content - ensures that the content of the file is exactly as specified
+                //                     refer to the Content message for more details
+                // -- minder.frizbee.actions.replace_tags_with_sha - finds any github actions within a workflow
+                //                                                   file and replaces the tag with the SHA using
+                //                                                   the frizbee library
+                string method = 3;
```

Comment:
any better suggestions than method?

Reply:
1. empty string defaults to `minder.content`, in other words defaults to the old method, the only one that was available previously. This was probably not super visible in the large patch I submitted initially, sorry about that, but it's this helper function: 
```go
func getMethod(prCfg *pb.RuleType_Definition_Remediate_PullRequestRemediation) string {
	if prCfg.Method == "" {
		return minderContentModification
	}

	return prCfg.Method
}
```
2. this is "after I had ingested the original git repo, what do I do to produce a remediation"? With this PR, I've added calling frizbee to what we had earlier, which was basically evaluate a text template and replace a file with the expanded template contents.